arch/x86/core/ia32/intstub.S

```diff
@@ -203,6 +200,12 @@ alreadyOnIntStack:
 	cli			/* disable interrupts again */
 #endif
 
+#if defined(CONFIG_TRACING_ISR)
+	pushl	%eax
+	call	sys_trace_isr_exit
```

Comment:
Does this need the `pushl/popl %edx` like above in line 183?

Reply:
no I don't believe so, it's unused after the call statement, we even zero it on line 211 if X2APIC is active

but we also don't need to push/pop EAX here either, note how it is zeroed on line 209